.circleci/config.yml

```diff
@@ -284,30 +284,28 @@ jobs:
       - image: cimg/base:2021.03
     steps:
       - echo_pipeline_parameters
-      - when:
-          condition:
-            or:
-              - equal: [<< pipeline.git.branch >>, master]
-              - equal: [<< pipeline.git.branch >>, production]
-          steps:
-            - run: |
-                curl --request POST \
-                  -u $CIRCLECI_API_TOKEN: \
-                  --url https://circleci.com/api/v2/project/github/CitizenLabDotCo/citizenlab/pipeline \
-                  --header 'content-type: application/json' \
-                  --header 'x-attribution-login: '"$CIRCLE_USERNAME" \
-                  --data '{"branch": "'"$CIRCLE_BRANCH"'","parameters": {"image_tag": "'"$CIRCLE_SHA1"'", "run_deploy": true, "back": true, "trigger": false}}'
+      - run: |
+          curl --request POST \
+            -u $CIRCLECI_API_TOKEN: \
+            --url https://circleci.com/api/v2/project/github/CitizenLabDotCo/citizenlab/pipeline \
+            --header 'content-type: application/json' \
+            --header 'x-attribution-login: '"$CIRCLE_USERNAME" \
+            --data '{"branch": "'"$CIRCLE_BRANCH"'","parameters": {"image_tag": "'"$CIRCLE_SHA1"'", "run_deploy": true, "back": true, "trigger": false}}'
 
   back-push-deployment-docker-image:
     resource_class: small
     docker:
       - image: cimg/base:2021.03
+    parameters:
+      image_tag:
+        type: string
+        default: << pipeline.parameters.image_tag >>
     steps:
       - echo_pipeline_parameters
       - setup_remote_docker
       - run: docker login -u $DOCKER_USER -p $DOCKER_PASS
-      - run: docker pull citizenlabdotco/back-ee:<< pipeline.parameters.image_tag >>
-      - run: docker tag citizenlabdotco/back-ee:<< pipeline.parameters.image_tag >> citizenlabdotco/back-ee:$CIRCLE_BRANCH
+      - run: docker pull citizenlabdotco/back-ee:<< parameters.image_tag >>
+      - run: docker tag citizenlabdotco/back-ee:<< parameters.image_tag >> citizenlabdotco/back-ee:$CIRCLE_BRANCH
```

Comment:
We could speed this up by updating the manifest using the docker registry API (instead of pulling-pushing). I did it in the past for cl2-nlp. There is also a new experimental `docker manifest` command that could be used instead of the API.

But this could be for another ticket :)

Reply:
That's great! I wondered about this but didn't know about it :)
Can you make a ticket for that?